docs/configuration.rst

```diff
@@ -160,6 +170,10 @@ creation. Otherwise defaults are assumed and only the specified user or group na
 `providers.cfg`_ and `permissions.cfg`_ for further details about specific formatting and behaviour of each available
 field.
 
+A section for webhooks has also been added to the combined configuration file. This section defines a list of
```

Comment:
Should add `versionadded:: 3.xxx` rst command to document when it becomes effective.
(not sure if it will be 3.5 or 3.6 depending on timing of other PR)